src/boot/z_std_dma.c

```diff
@@ -405,10 +399,10 @@ void DmaMgr_ProcessRequest(DmaRequest* req) {
                     DMA_ERROR(req, filename, "Segment Alignment Error",
                               T("セグメント境界をまたがってＤＭＡ転送することはできません",
                                 "DMA transfers cannot cross segment boundaries"),
-                              "../z_std_dma.c", LN3(575, 578, 726));
+                              "../z_std_dma.c", LN3(575, 578, 726, 595));
```

Comment:
It would be nice to put iQue before Gamecube, that way it would be in ascending order